packages/start-vercel/index.js

```diff
@@ -82,6 +82,57 @@ export default function ({ edge, prerender } = {}) {
           };
       writeFileSync(join(renderFuncDir, ".vc-config.json"), JSON.stringify(renderConfig, null, 2));
 
+      // Generate API function
+      const apiRoutes = config.solidOptions.router.getFlattenedApiRoutes()
+      if (apiRoutes.length > 0) {
+        let baseEntrypoint = "entry.js";
+        if (edge) baseEntrypoint = "entry-edge.js";
+        copyFileSync(join(__dirname, baseEntrypoint), entrypoint);
+
+        const bundle = await rollup({
+          // Same as render
+          input: entrypoint,
+          plugins: [
+            json(),
+            nodeResolve({
+              preferBuiltins: true,
+              exportConditions: edge ? ["worker", "solid"] : ["node", "solid"]
```

Comment:
Maybe there should be a separate option in the config for using edge API routes.

A better solution would definitely be having it on a [per function basis](https://nextjs.org/docs/api-routes/edge-api-routes#basic), but that would require regenerating a new file for each function.